compiler/rustc_codegen_ssa/src/back/symbol_export.rs

```diff
@@ -127,7 +127,10 @@ fn reachable_non_generics_provider(tcx: TyCtxt<'_>, _: LocalCrate) -> DefIdMap<S
                     || codegen_attrs.flags.contains(CodegenFnAttrFlags::USED_LINKER),
                 rustc_std_internal_symbol: codegen_attrs
```

Comment:
can you add a FIXME to `SymbolExportInfo::rustc_std_internal_symbol` that the field is also used for eii?